### DISCUSSION TITLE
Withdrawal coin problems

### DISCUSSION THREAD

**Discussion by NenadJ1983** [Chia-Network/chia-blockchain#7765] [Support]
Your Honor, I have a problem with the withdrawal of coins on the chia wallet.a transaction made, however, does not come for several days. I'm asking for your help[](url)

**Comment by keliew** [comment]
Your Highness, some details of the process/procedure along with an idea about your setup, and/or output of logs would be helpful.

If you wish to withdraw to gold, it is unfortunately, not possible in this era.

**Reply by NenadJ1983** [reply]
My chia walet doesn't see the transaction

**Comment by NenadJ1983** [comment]
![Picture1](https://user-images.githubusercontent.com/87868715/126870210-4380afe5-bb75-4c0d-b7b5-f9e1c09a3227.png)

**Comment by NenadJ1983** [comment]
![Picture2](https://user-images.githubusercontent.com/87868715/126870261-017f5983-8722-4695-b780-1243fc11830d.png)

**Comment by NenadJ1983** [comment]
please, if you have advice or a solution, I'd appreciate it

**Comment by marssystems** [comment]
Your wallet isn't synced with the blockchain. Wait until it has synced and then you will see your transaction.

**Reply by NenadJ1983** [reply]
but transaction is on two weeks

**Comment by NenadJ1983** [comment]
Is there any way I can try synchronizing again or do something

**Comment by NenadJ1983** [comment]
I'm grateful for the response and the effort

**Comment by marssystems** [comment]
That doesn't matter. You have to sync your wallet to see it.

**Comment by NenadJ1983** [comment]
Can you help me with advice on how to synchronize my wallet

**Comment by NenadJ1983** [comment]
![Picture3](https://user-images.githubusercontent.com/87868715/126870547-4c9c4153-c632-455e-9bb1-e2cfbad6e397.png)

**Comment by NenadJ1983** [comment]
on the node shows that it is synchronized, on the wallet not

**Comment by NenadJ1983** [comment]
I'm once again scouting for the time off , and grateful if you have a proposal for a solution

**Comment by marssystems** [comment]
Shut down chia.exe then delete the wallet.db file. Then restart chia.exe and the wallet should re-sync.

**Comment by NenadJ1983** [comment]
I thank you, I will follow your instructions
